workers/src/main/java/edu/cornell/repository/ConfigParser.java

```diff
@@ -0,0 +1,90 @@
+package edu.cornell.repository;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.EnumMap;
+import java.util.List;
+
+/**
+ * Parses the config file of the repository.
+ */
+public class ConfigParser {
```

Comment:
In the class comment, can you add documentation on what the syntax of the config file is? I'm struggling to understand how this one is laid out and parsed. Can you maybe add an example config file into this package as well for reference?